install.sh

```
@@ -212,6 +220,11 @@ apt)
   install_pkg libsecret-1-dev
   install_pkg libsecret-tools
   ;;
+  ;;
```

Shellcheck (0.11.0)

[error] Expected 'esac' to close the case statement. Fix any mentioned problems and try again.

(SC1072)